docs/eog/eog-in-practice/cw21/event-project-stage/eps-agenda.rst

```diff
@@ -114,14 +114,14 @@ A schedule was scaffolded so that participants could work around suggested break
 
 Each Hack Day team had five minutes to demonstrate their projects and how they met the judging criteria:
 
-1. Novelty, creativity, coolness and/or usefulness
-2. Implementation and infrastructure 
-3. Demo and presentation
-4. Project transparency
-5. Future potential
-6. Team work
+1. Novelty, creativity, 'coolness' and/or usefulness.
+2. Implementation and infrastructure. 
+3. Demo and presentation.
+4. Project transparency.
+5. Future potential.
+6. Team work.
```

Comment:
```suggestion
6. Team work
```

Reply:
Again due to - https://github.com/softwaresaved/event-organisation-guide/blob/master/STYLE.md#bulleted-lists-punctuation - we consistently have followed this so changing would mean changing a lot of other parts of EOG